quinn/src/builders.rs

```diff
@@ -100,6 +103,15 @@ where
         self.default_client_config = config;
         self
     }
+
+    /// Use a customized cid generator factory in the endpoint
+    pub fn use_customized_cid<F: Fn() -> Box<dyn ConnectionIdGenerator> + Send + Sync + 'static>(
```

Comment:
```suggestion
    pub fn connection_id_generator<F: Fn() -> Box<dyn ConnectionIdGenerator> + Send + Sync + 'static>(
```
For builder methods, preferred naming is a noun specifying the property being modified.

Reply:
(fixing review status)